subschemas/substrate-chain/src/typeDefs/account.ts

```diff
@@ -43,13 +43,25 @@ export default /* GraphQL */ `
     account: Account!
   }
 
+  type Voter {
+    address: String!
+    account: Account!
+  }
+
+  type CouncilVote {
+    stake: String!
+    formattedStake: String!
+    votes: [Voter!]!
+  }
+
   type Account {
     address: String!
     display: String!
     hasIdentity: Boolean!
     registration: DeriveAccountRegistration!
     balance: AccountBalance!
     subAccounts: [SubAccount!]
+    councilVote: CouncilVote
```

Comment:
As convenient as this could be, it does not belong here. You can extend `Account`  and add this field to `Voter`, also add it's own resolver